api/endpoints.go

```diff
@@ -195,6 +195,8 @@ func get(i *jsonAPIHandler, path string, w http.ResponseWriter, r *http.Request)
 		i.GETWalletStatus(w, r)
 	case strings.HasPrefix(path, "/ob/ipns"):
 		i.GETIPNS(w, r)
+	case strings.HasPrefix(path, "/ob/resolveipns"):
+		i.ResolveIPNS(w, r)
```

Comment:
Core function `ResolveIPNS` does not follow pattern of `HTTPVERBFunctionName`. Nitpick: These two function calls are a little ambiguous to a newcomer. Any way to clarify?

Reply:
You're right, I totally overlooked the http verb prefix thing. I'll fix that.

Reply:
I'm not sure how to clarify without renaming /ob/ipns, which would not be ideal.

One option would be making it a flag on /ob/ipns, e.g. /ob/ipns?resolve=true

Reply:
I can't think of any suggestions to offer either. We can punt if we can't think of anything better. I think the GET param overloads the endpoint with two different behaviors. And really, I would want to change the `ipns` endpoint to be more specific, but that has repercussions which are annoying. We can leave it as-is for now.